source/Visibility/ProAppVisibilityModule/Helpers/FeatureClassHelper.cs

```diff
@@ -290,28 +290,64 @@ public class FeatureClassHelper
         /// <param name="simplify"></param>
         /// <param name="rasterField"></param>
         /// <returns></returns>
-        public static async Task IntersectOutput(string inputRasterLayer, string outputPolygonLayer, bool simplify, string rasterField)
+        public static async Task IntersectOutput(string inputRasterLayer, string outputPolygonLayer, 
+            bool simplify, string rasterField, string intersectMaskFeatureClass)
         {
+            string rasterToPolyLayer = outputPolygonLayer;
```

Comment:
Will do - I also meant (but forgot) to change that debug to a message box below so better user feedback if/when tool fails